README.md

```diff
@@ -10,36 +10,98 @@ If you want to contribute to this project, there are several things that you hav
 2. Exactly Java 1.8 
 3. MySQL & Tomcat
 4. Gradle
-5. Slack account
+5. Slack/Telegram/Facebook account
+
+And for NLU, you are required to have
+1. Python 3.6.1 (as NOW Rasa NLU is having dependency issues with Python 3.7)
+2. Microsoft VC++ Compiler
+3. Rasa (for offline training and testing models)
+
+```
+# install Rasa NLU using these commands
+
+pip install rasa[spacy]
+python -m spacy download en_core_web_md
+python -m spacy link en_core_web_md en
+```
 
 ## Getting started
 
-### Installing on Mac OS
-Java: <br>
+### Setting up
+##### Java:
 There are two options. 
 1. Download Java 1.8 from [official website](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html), install it and configure it on class path
 2. If you have `brew` installed in your system, you can also download from homebrew. `brew cask install java8` to install Java 8 on Mac OS. 
 
-MySQL:
+##### MySQL:
 1. Download from [official website](https://dev.mysql.com/downloads/mysql/), install it and configure it. During installation, if the user does not set the initial password, then the initial password should be set in the System Preferences -> MySQL -> set initial password. Then configure mysql in the classpath. 
+1.1. If you want to use Cloud Service for database, you are free to do that. Remember to update `application.properties` file. 
+2. Create table `user_creds` using following MySQL command
+```
+create table users_creds
+(
+	username varchar(255) not null,
```

Comment:
Can potentially look into encrypting these fields at the database level for when we want to extend this work.